src/components/sbb-navigation-action/sbb-navigation-action.tsx

```diff
@@ -9,11 +9,18 @@ import {
   resolveRenderVariables,
 } from '../../global/interfaces/link-button-properties';
 import { documentLanguage, SbbLanguageChangeEvent } from '../../global/helpers/language';
+import { hostContext } from '../../global/helpers/host-context';
+import { AgnosticMutationObserver as MutationObserver } from '../../global/helpers/mutation-observer';
+import { isValidAttribute } from '../../global/helpers/is-valid-attribute';
 
 /**
  * @slot unnamed - Use this slot to provide the navigation action label.
  */
 
+const navigationActionObserverConfig: MutationObserverInit = {
+  attributeFilter: ['active'],
```

Comment:
Why don't we just add an "@Prop" for active? and then watch changes?

Reply:
We already have an `active` prop, but I do so for the same reason as in the toggle, to avoid getting into a loop: https://github.com/lyne-design-system/lyne-components/pull/1481#discussion_r1064444502

Reply:
ah sorry :D I understand! Can we document somehow this approach?

Reply:
I added a comment on the callback as in the toggle. You tell me if you'd like to be more specific.